src/classes/CourseConSettingHealthCheckVMapper.cls

```diff
@@ -57,71 +56,82 @@ public virtual with sharing class CourseConSettingHealthCheckVMapper {
-        healthCheckItemList.add(hcItem3);
-
-        HealthCheckGroupVModel hlthChkGroupVMod = new HealthCheckGroupVModel(hlthCheckGroupLabel,
-                                                                             hlthChkGroupStatus,
-                                                                             hlthChkGroupPassedChecks,
-                                                                             hlthChkGroupTotalChecks,
-                                                                             hlthChkGroupExpandedRowsList,
-                                                                             healthCheckItemList);
+        HealthCheckGroupVModel courseConnEnableCheckResult = this.getCourseConnectionsNotEnabledHealthCheckItems(courseConnSettingsModel);
 
+        hlthChkGroupVMod.passedChecks += courseConnEnableCheckResult.passedChecks;
+        hlthChkGroupVMod.totalChecks +=  courseConnEnableCheckResult.totalChecks;
+        hlthChkGroupVMod.expandedRowsList.addAll(courseConnEnableCheckResult.expandedRowsList);
+        hlthChkGroupVMod.healthCheckItemList.addAll(courseConnEnableCheckResult.healthCheckItemList);
+
+        if(hlthChkGroupVMod.passedChecks < hlthChkGroupVMod.totalChecks) {
+            hlthChkGroupVMod.status = Label.stgHealthCheckStatusFailed;
+        }
 
         hlthChkGroupVMod.healthCheckItemList.sort();
         return hlthChkGroupVMod;
 
     }
-    
+
+    /*******************************************************************************
+    * @description Retrieves the health check for a Course Connection Settings Model
+    * @param CourseConnectionSettingsModel The courseConnSettingsModel model to perform
+    * the health check on.
+    * @return The health check for the Course Connection Settings Model
+    *******************************************************************************/
+    @TestVisible
+    protected virtual HealthCheckGroupVModel getCourseConnectionsNotEnabledHealthCheckItems(
+        CourseConnectionSettingsModel courseConnSettingsModel
+    ) {
+
+        HealthCheckGroupVModel healthCheckGroupVModel = new HealthCheckGroupVModel(
+            '',
+            '',
+            0,
+            0,
+            new List<String>(),
+            new List<HealthCheckItemVModel>()
+        );
+
+        if (courseConnSettingsModel.courseConnectionsEnabled == false) {
+            healthCheckGroupVModel.healthCheckItemList.add(new HealthCheckItemVModel('01',
+                                                                      'Label.stgEnableCourseConnectionsTitle',
+                                                                      'Fail',
+                                                                      Label.stgHealthCheckStatusFailed,
+                                                                      'Label.stgHCCourseConEnable',
+                                                                      'Label.stgHCCourseConEnableFix',
```

Comment:
The labels themselves evaluate to strings. By placing them within single quotes they are evaluating to the literals and not able to be localized. Always check your UI visually when implementing UI.
```suggestion
                                                                      Label.stgEnableCourseConnectionsTitle,
                                                                      'Fail',
                                                                      Label.stgHealthCheckStatusFailed,
                                                                      Label.stgHCCourseConEnable,
                                                                      Label.stgHCCourseConEnableFix,
```

Reply:
Ah! Didn't realize these were merged in, will edit them.